src/Reflex/Dom/Contrib/Widgets/ButtonGroup.hs

```diff
@@ -167,15 +167,15 @@ radioGroup dynName dynEntryList cfg = do
      <> bool mempty ("checked" =: "checked") chkd
     handleOne _ dynV dynChecked = do
 
-      el "tr" $ do
+      el "tr" $ el "td" $ el "label" $  do
         let txt = zipDynWith (\v m -> fromMaybe "" $ Prelude.lookup v m)
                              dynV dynEntryList
 
             btnAttrs = mkBtnAttrs <$> dynName <*> dynChecked
-        (b,_) <- el "td" $ elDynAttr' "input" btnAttrs $ return ()
+        (b,_) <- elDynAttr' "input" btnAttrs $ return ()
         f <- holdDyn False $ leftmost [ False <$ (Blur  `domEvent` b)
                                       , True  <$ (Focus `domEvent` b)]
-        el "td" $ dynText txt
+        el "text" $ dynText txt
```

Comment:
Is this a valid HTML tag?  I'm not familiar with it and a cursory Google search came up empty.  Could we use a `<span>` here instead?